2/test.js

```diff
@@ -0,0 +1,46 @@
+// Import the mount() method from the test utils
+// and the component you want to test
+import { mount } from '@vue/test-utils'
+import ListItem from './ListItem'
+
+const wrapper = mount(ListItem)
+
+describe('given a `todo`', () => {
+    it('renders todo text', () => {
```

Comment:
```suggestion
    it('renders save button', () => {
```
Would fit better to the implementation of the test case.